src/Mesh_TEST.cc

```diff
@@ -140,6 +166,21 @@ TEST(DOMMesh, Set)
   sdf::Mesh mesh;
   EXPECT_EQ(nullptr, mesh.Element());
 
+  EXPECT_EQ(std::string(), mesh.OptimizationStr());
+  mesh.SetOptimization("convex_hull");
+  EXPECT_EQ("convex_hull", mesh.OptimizationStr());
+  EXPECT_EQ(sdf::MeshOptimization::CONVEX_HULL, mesh.Optimization());
+  mesh.SetOptimization(sdf::MeshOptimization::CONVEX_DECOMPOSITION);
+  EXPECT_EQ("convex_decomposition", mesh.OptimizationStr());
+  EXPECT_EQ(sdf::MeshOptimization::CONVEX_DECOMPOSITION,
+            mesh.Optimization());
```

Comment:
```suggestion
            mesh.Optimization());
  EXPECT_FALSE(mesh.SetOptimization("invalid"));
```
improve coverage a little

Reply:
[f82be1f](https://github.com/gazebosim/sdformat/pull/1382/commits/f82be1f92390c6b2795bd18ae49875fb6ddb638a)